src/Common/src/System/Security/Cryptography/Asn1/AlgorithmIdentifierAsn.xml.cs

```diff
@@ -38,11 +39,11 @@ internal static AlgorithmIdentifierAsn Decode(ReadOnlyMemory<byte> encoded, AsnE
         {
             return Decode(Asn1Tag.Sequence, encoded, ruleSet);
         }
-
+        
         internal static AlgorithmIdentifierAsn Decode(Asn1Tag expectedTag, ReadOnlyMemory<byte> encoded, AsnEncodingRules ruleSet)
         {
             AsnReader reader = new AsnReader(encoded, ruleSet);
-
+            
```

Comment:
You seem to be adding a lot of positive whitespace...

Reply:
Was this just something that got cleaned up previously?

Reply:
> Was this just something that got cleaned up previously?

Yes, I previously removed all trailing whitespace from across corefx and coreclr and enabled the stylecop rule to flag it.  The .xslt is regenerating that whitespace unfortunately, which is why I added the suppress pragma; tweaking the xslt to avoid generating all that whitespace didn't seem worthwhile, but if you think it'd be easy, I'd be happy to be able to remove the suppressions.

Reply:
It's probably 20-40 minutes of annoying fiddling with increased complexity and low return.  I agree with suppress-and-ignore.  (My initial thought had been that your edit of the xslt had introduced the whitespace, but no, it was my creation of it in the long ago :smile:)

Reply:
:smile: